src/features/pages/trade-types/options/digital/digits/data.ts

```diff
@@ -0,0 +1,254 @@
+import DigitsMatchesDiffers from 'images/svg/trade-types/options/digital/digits-matches-differs.svg'
```

Comment:
Hi @meenakshi-deriv When we have same folder with lots of images, let's follow this
```
import {
  DigitsMatchesDiffers,
  DigitsEvenOdd,
  DigitsOverUnder,
  .......
  UnderLossIcon,
} from 'images/svg/trade-types/options/digital';
```

Reply:
Updated @habib-deriv , thanks